ansible/roles/kafka/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+apache_mirror: apache.mirror.anlx.net
+kafka_version: 0.8.2.2
+kafka_user: kafka
+kafka_group: kafka
+kafka_data_dir: /opt/data/kafka/data
+kafka_conf_dir: /etc/kafka/config
+kafka_log_dir: /var/log/kafka
```

Comment:
not sure if this is right or it should go in `{{ log_home }}` (looked like I would have had to wrangle with permissions to make that work so just did this)